README.md

```diff
@@ -8,6 +8,10 @@
 [![GitHub forks](https://img.shields.io/github/forks/microsoft/IoT-For-Beginners.svg?style=social&label=Fork)](https://GitHub.com/microsoft/IoT-For-Beginners/network/)
 [![GitHub stars](https://img.shields.io/github/stars/microsoft/IoT-For-Beginners.svg?style=social&label=Sta)](https://GitHub.com/microsoft/IoT-For-Beginners/stargazers/)
 
+[![Bengali](https://img.shields.io/badge/-Bengali-blue)](https://github.com/microsoft/IoT-For-Beginners/blob/main/translations/README.bn.md)
```

Comment:
Can these links be relative paths instead please.

Reply:
> Can these links be relative paths instead please.

Hi I'm making the relative changes . I have a question, would you want me to add the shields to the English readme only or for every translated main readme?

Reply:
GitHub has a handy button to do this!

https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/syncing-a-fork

Reply:
I would add the shields to all the top level README files, so that if someone shares a link to a readme in one language, it's easy for folks to jump to a different one if needed.

Reply:
> I would add the shields to all the top level README files, so that if someone shares a link to a readme in one language, it's easy for folks to jump to a different one if needed.

Sure, will do !